lib/index.js

```diff
@@ -90,16 +85,25 @@ const format = string => date =>
  */
 const family = (file, files) => {
   const ret = {};
-  let dir = dirname(file);
+  let dir = path.dirname(file);
 
-  if ('.' == dir) {
+  if (dir === '.') {
     dir = '';
   }
 
   for (const key in files) {
-    if (key == file) continue;
-    if (key.indexOf(dir) != 0) continue;
-    if (html(key)) continue;
+    if (key === file) {
+      continue;
+    }
+
+    if (key.indexOf(dir) !== 0) {
+      continue;
+    }
+
+    if (html(key)) {
+      continue;
+    }
```

Comment:
I guess it's sorta a matter of personal preference, but expanding these one-line conditionals to multiple lines seems unnecessary. (Unless this was breaking an eslint rule or something?)

Reply:
Not sure anymore, but I'm willing to try to revert these later after we move with my other patches. Because I'm mainly on Windows, the LF issue is bugging me a lot :P